### PR TITLE
Multiple upload folders

### PR DESCRIPTION
**Pull request by Ph0tonic**
Add possibility to upload to multiple folders

### CODE REVIEW

File: downloadservice/app.py [hunk 1]
```diff
@@ -360,8 +364,25 @@ def upload(user, path):
     if '..' in path:
         return "Error: path cannot contain '..'", 400
 
+    # check if upload folder is accessible
+    potential_folders = app.config['CTADS_UPSTREAM_UPLOAD_FOLDERS']
+    selected_base_folder = None
+    for base_folder in potential_folders:
+        try:
+            _, status_code = list(path=urljoin_multipart(base_folder, 'users'))
```
Comment (volodymyrss) [inline downloadservice/app.py:372]:
```suggestion
            _, status_code = _list(path=urljoin_multipart(base_folder, 'users'))
```

Comment (volodymyrss) [reply downloadservice/app.py:372]: we should really rename the `list` which is reserved

**Review by volodymyrss — APPROVED** [review]

File: downloadservice/app.py (hunk 1 above)
Comment (Ph0tonic) [reply downloadservice/app.py]: Renamed, but not with with an underscore as this as the meaning of internal use in python.

**Review by volodymyrss — APPROVED** [review]
And yes, it seems to work: I can upload CTA and LST dir depending on the certificate.

**Comment by Ph0tonic** [comment]
@volodymyrss I pushed on gitlab to merge it there otherwise I have not rights to push the master branch.
- https://gitlab.cta-observatory.org/cta-computing/suss/sdc-portal/prototypes/downloadservice/-/merge_requests/12

**Review by volodymyrss — APPROVED** [review]

**Pull request merged**